Makefile

```diff
@@ -539,3 +539,11 @@ run-rp: ci-rp podman-secrets
 		--secret proxy-client.crt,target=/app/secrets/proxy-client.crt \
 		--secret proxy.crt,target=/app/secrets/proxy.crt \
 		$(LOCAL_ARO_RP_IMAGE):$(VERSION) rp
+
+.PHONY: build-selenium
+build-selenium:
+	docker compose build selenium
```

Comment:
Since we aren't referencing a `.services.selenium.build` I don't think this command does anything. Can we remove it?

Reply:
Thanks for pointing me.
Remove unused build-selenium target from Makefile as no build configuration exists.